Source/lighting.cpp

```diff
@@ -391,7 +393,8 @@ void MakeLightTable()
 					// Leaner falloff
 					scaled = factor * maxDarkness;
 				}
-				LightFalloffs[radius][distance] = static_cast<uint8_t>(scaled + 0.5F); // round up
+				scaled += 0.5F; // Round up
+				LightFalloffs[radius][distance] = static_cast<uint8_t>(scaled);

```

Comment:
Was this a problem for VC 2019?
I would prefer we don't start modifying values if we do not need to.